ui/perfherder/CompareSelectorView.jsx

```diff
@@ -210,7 +221,7 @@ export default class CompareSelectorView extends React.Component {
                   color="info"
                   className="mt-2 mx-auto"
                   onClick={
-                    newRevision !== '' && disableButton ? '' : this.submitData
+                    newRevision !== '' && disableButton ? null : this.submitData
```

Comment:
I know this is pre-existing.  But it seems like it would be better, if this criteria came back true, then the button should be grayed out and a tooltip saying why, rather than just ``null``ifying the ``onClick``.  But that could be in a later PR.

Reply:
Oh ok, I think I follow.  Sounds good.  :)